docs/examples/custom.ipynb

```diff
@@ -25,6 +25,7 @@
     "import numpy as np\n",
     "import matplotlib.pyplot as plt\n",
     "import pandas as pd\n",
+    "%matplotlib inline\n",
```

Comment:
hi @sangfrois, apparently, according to a discussion here https://github.com/spatialaudio/nbsphinx/issues/447#issuecomment-630766064, the line `%matplotlib inline` isn't needed anymore in modern IPython versions. Did it not work for you?

Reply:
Hi @Tam-Pham, so the problem is that you get a Javascript error in the Binder environment. ``%matplotlib inline`` fixes that problem in the cloud examples. So, yes I can run the notebooks without this command on my local machine, but not using the link in the README under ``docs/examples".

Does that make any sense ? :)